packages/react-query/src/useSuspenseInfiniteQuery.ts

```diff
@@ -31,50 +29,16 @@ export interface UseSuspenseInfiniteQueryOptions<
  * This hook is wrapping useInfiniteQuery of @tanstack/react-query v4 with default suspense option.
  * @see {@link https://suspensive.org/docs/react-query/useSuspenseInfiniteQuery}
  */
-// arg1: queryKey, arg2: queryFn, arg3: options
-export function useSuspenseInfiniteQuery<
-  TQueryFnData = unknown,
-  TError = unknown,
-  TData = TQueryFnData,
-  TQueryKey extends QueryKey = QueryKey,
->(
-  queryKey: TQueryKey,
-  queryFn: QueryFunction<TQueryFnData, TQueryKey>,
-  options?: Omit<UseSuspenseInfiniteQueryOptions<TQueryFnData, TError, TData, TQueryKey>, 'queryKey' | 'queryFn'>
-): UseSuspenseInfiniteQueryResult<TData, TError>
-// arg1: queryKey, arg2: options
-export function useSuspenseInfiniteQuery<
-  TQueryFnData = unknown,
-  TError = unknown,
-  TData = TQueryFnData,
-  TQueryKey extends QueryKey = QueryKey,
->(
-  queryKey: TQueryKey,
-  options: Omit<UseSuspenseInfiniteQueryOptions<TQueryFnData, TError, TData, TQueryKey>, 'queryKey'>
-): UseSuspenseInfiniteQueryResult<TData, TError>
-// arg1: options
```

Comment:
From @suspensive/react-query v2, there is only one parameter to pass infiniteQueryOptions